app/src/main/kotlin/cz/covid19cz/erouska/jobs/AutoRestartJob.kt

```diff
@@ -0,0 +1,84 @@
+package cz.covid19cz.erouska.jobs
+
+import android.app.AlarmManager
+import android.app.PendingIntent
+import android.content.BroadcastReceiver
+import android.content.Context
+import android.content.Intent
+import android.net.Uri
+import android.os.Handler
+import cz.covid19cz.erouska.BuildConfig
+import cz.covid19cz.erouska.service.CovidService
+import cz.covid19cz.erouska.utils.L
+import org.threeten.bp.ZonedDateTime
+
+class AutoRestartJob : BroadcastReceiver() {
```

Comment:
Prejmenovat na AutoRestartReceiver?

Reply:
Já jsem tak trochu zavedl novou jmennou konvenci :smile:  
Nevím, jestli takových případů bude víc (nebo už je :scream:) ale pojmenoval jsem to `*Job` (a je to v package `jobs`, jelikož je to prostě nějakej naplánovanej úkol a chtěl jsem to odlišit od ostatních receiverů; přičemž to, že je tohle receiver, je jen interní věc toho jobu, ale zbytek appky to vůbec nemusí vědět.  
Ale jako jestli jsem si troufnul moc :smile: nebo je to kokotina, tak to klidně změním. Je to trochu kreativní a možná subjektivní řešení :wink:

Reply:
Rozumim. Muj pohled na to je, ze BR jsou systemovy komponenty, ktery maji prijimat eventy. Normalne instance vytvari a managuje system, rucne to delas jen ve chvili kdy ho explicitne registrujes. Ja to tedy beru jako neco, kde bys nemel mit moc kodu a zadny stav. Job klidne mit muze, ale jak jsem psal jinde, dal bych to do extra tridy.